test/MuxManagerTest.cpp

```diff
@@ -924,7 +924,7 @@ INSTANTIATE_TEST_CASE_P(
     MuxState,
     GetMuxStateTest,
     ::testing::Values(
-        std::make_tuple("active", mux_state::MuxState::Label::Active),
+        std::make_tuple("active", mux_state::MuxState::Label::Wait),
```

Comment:
Can you please help me understand the difference between the Wait and Active?

Reply:
`Wait` is when linkmgrd writes to APP_DB, it will "wait" for the toggle/muxprobe result, so it will have the latest hw state. 

`Active` is simply as it implies. 

The unit test here is to validate how an **active** port reacts to STATE_DB value == "active". It used to be no-op, so linkmgrd state remains "Active", but now we are enforcing another toggle, so it will be "Wait".

Reply:
ok, so can I assume `Standby `is already handled in this flow? Since you added test for only `Active`

Reply:
> ok, so can I assume `Standby `is already handled in this flow? Since you added test for only `Active`

Yes.